src/main/resources/freemarker/new-library-card-issued.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Broad Data Use Oversight System - Your Library Card Has Been Issued!</title>
```

Comment:
```suggestion
  <title>Broad Data Use Oversight System - Your Library Card Has Been Issued</title>
```

Consider a more neutral tone.